docs/configuration/behavior/index.mdx

```diff
@@ -230,6 +231,22 @@ For more information, see the [Kubernetes docs](https://kubernetes.io/docs/conce
 
 ---
 
+## Customize warnings (logs)
+
+By default, Datree prints a warning for the following types of events:
+* **Passed policy checks** - resources that passed the policy check. Helm chart param: `passedPolicyCheck`.
```

Comment:
I think that using `Helm chart param` and afterwards in `The Helm chart parameter `datree.enabledWarnings`` is a bit confusing. For me it doesn't explains that `datree.enabledWarnings` is an array that can contain the strings you mentioned